index.html

```diff
@@ -84,20 +84,23 @@
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
           <!-- Added Twitter share button, link share only-functionality per Issue 3 -->
```

Comment:
Perhaps we should increase button sizes for all share buttons (lines 85-87, 95, 103) by 25% to increase page visibility and contrast from content. 25%= 18px or 1.125rem?